### DISCUSSION TITLE
RealtekAudioControl_1.32.261-offline

### DISCUSSION THREAD

**Discussion by joseluisfloressanchez** [pal1000/Realtek-UAD-generic#97] [Q&A] [answered]
Error al instalar el paquete offline, no es la version 1.32.261.

https://ibb.co/jzdKkfR

https://ibb.co/9r4QNYg

https://ibb.co/XbW2mn7

**Comment by pal1000** [comment, marked as answer]
It looks like [Gigabyte OEM package](https://download.gigabyte.com/FileList/Driver/nb-driver-64bit-win10-win11-dchu-audio-6.0.9282.1.zip) from which I created my generic package bundled an older RTKUWP offline package.  Sorry I didn't pay enough attention and let myself duped like this. I'll fix it really soon.

Update: Fixed it in [RealtekAudioControl_1.32.261-offline-r2.zip](https://github.com/pal1000/Realtek-UAD-generic/releases/download/6.0.9282.1/RealtekAudioControl_1.32.261-offline-r2.zip)

**Comment by joseluisfloressanchez** [comment]
Muchas gracias por su gran trabajo!!!!!!
